demo_calibration.py

```
@@ -0,0 +1,266 @@
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from datetime import datetime, timedelta
+import sys
+import os
+
+# Add backend to path
+sys.path.append(os.path.join(os.path.dirname(__file__), 'backend'))
+
+from backend.core.CalibrationEngine import CalibrationEngine, CalibrationMetrics
```

Ruff (0.12.2)

`backend.core.CalibrationEngine.CalibrationMetrics` imported but unused

Remove unused import: `backend.core.CalibrationEngine.CalibrationMetrics`

(F401)